network/datadog_checks/network/network.py

```diff
@@ -433,6 +433,29 @@ def _check_linux(self, instance):
                     self._submit_netmetric(nstat_metrics_names[k][met], self._parse_value(netstat_data[k][met]),
                                            tags=custom_tags)
 
+        proc_conntrack_path = "{}/net/nf_conntrack".format(proc_location)
+        try:
+            with open(proc_conntrack_path, 'r') as conntrack_file:
+                # Starting at 0 as the last line has a line return
+                conntrack_count = 0
+                while 1:
+                    conntrack_buffer = conntrack_file.read(65536)
```

Comment:
Why `65536`?

Reply:
It just reads the file by chunks of 64k to avoid loading too much data into memory at once. Random choice. Do you have a preference in terms of buffer size?

Reply:
Done.